composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "adldap2/adldap2-laravel",
+    "name": "minkbear/adldap2-laravel",
```

Comment:
Don't do that. You should use composer's repositories for that.

https://getcomposer.org/doc/05-repositories.md#loading-a-package-from-a-vcs-repository